docs/adr/components-for-policy-scanning.md

```diff
@@ -0,0 +1,55 @@
+# Component Versioning for Openstack Entities
+
+## Context and Problem Statement
+
+How to accomodate Openstack entities like Security Groups in the existing Entity Relationship of Heureka? 
+
+The current Entity Relationships in Heureka are defined for vulnerabilities in objects like containers, where it is intuitive to have ComponentVersion and ComponentInstances. For non-compliance (not exactly vulnerabilities) in entities like Security Group, how to adapt to existing data model?
+
+## Decision Drivers
+
+* Fits into existing data model 
+* Make use of Heureka Issue Matching through Component Version
+
+## Considered Options
+
+* Opt 1: Using the configuration as ComponentVersion
+* Opt 2: Adding ComponentContext to ComponentInstance
+* Opt 3: Do not store any configuration context data
+
+## Decision Outcome
+
+Chosen option: "Opt 1: Using Configuration as ComponentVersion", because neither option 2 or 3 fully utilize existing entity relationships in Heureka
+
+### Consequences
+
+* Good, because we can decrypt the hash to get the configuration back to do processing
```

Comment:
Well a hash can not be decrypted. 

If we use an HMAC we could but probably we gonna store a Hash.

Please remove that "pro"

Reply:
removed